### ISSUE TITLE
Collaboration

### ISSUE THREAD

**Issue by zsviczian** [zsviczian/obsidian-excalidraw-plugin#408]
https://github.com/excalidraw/excalidraw/discussions/3879#discussioncomment-1110524

**Comment by zsviczian** [comment]
I know it can be done...
https://github.com/excalidraw/excalidraw/discussions/4969

The first attempt to integrate CollabWrapper into the Excalidraw package was not successful. 
https://github.com/zsviczian/excalidraw/blob/for-obsidian-collab/src/packages/excalidraw/index.tsx

Next I will see if there is an easy way of integrating

**Comment by thibaultmol** [comment]
As someone who's looking into using obsidian for my small business, this would be a massive addition!

**Comment by zsviczian** [comment]
@thibaultmol : note that Obsidian already offers shared Vaults. For a small business of up to 10 people I guess that is a workable option. My plugin supports parallel editing in a shared Vault using Obsidian sync to facilitate the collaboration. Note, that it is not real time collaboration like on Excalidraw.com, but depending on the speed of sync there could be few seconds of delays.... however it should still work reasonably well.

**Comment by thibaultmol** [comment]
My plan was to use Obsidian sync (or the obsidian livesync plugin, have to checkout how well it works though). I don't mind paying for a good service (and the business license itself of course) but actual realtime editing ... is kind of a must in my opinion.

So this issue is more for people that want to sync an excalidraw but not sync the vault using official Sync?

**Comment by zsviczian** [comment]
Yes, synchronizing only Obsidian-Excalidraw without synchronizing the vault
is not possible currently, unless you use excalidraw.com and embed the
excalidraw collaboration room as an iFrame (see this discussion here:
https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/724#issuecomment-1199120276
)

If you synchronize the entire Vault, then it should work... best is to give
it a try. I would be very interested to hear about your experiences.
